src/components/ContentSidebar/MetadataSidebar.js

```diff
@@ -212,23 +236,113 @@ class MetadataSidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     onModification = (id: string, isDirty: boolean) => {
-        const { editors = [] }: State = this.state;
-        const index = editors.findIndex(({ instance }) => instance.id === id);
-        if (index === -1) {
+        const oldEditor = this.getEditor(id);
+        if (!oldEditor) {
```

Comment:
Do we need to check `!file` here, as well?

Reply:
Not needed here since file isn't used. Also function isn't accessible if no file. Other places also the same holds true, but to keep flow happy null check is there